proto/substrait/algebra.proto

```diff
@@ -891,6 +911,12 @@ message Expression {
     }
   }
 
+  enum WindowFrame {
+    WINDOW_FRAME_UNSPECIFIED = 0;
+    WINDOW_FRAME_ROWS = 1;
+    WINDOW_FRAME_RANGE = 2;
```

Comment:
Maybe add `WINDOW_FRAME_GROUPS` for peer groups?

Reply:
The definition of `WindowFrame ` refers to the definition of Window in [Spark ](https://spark.apache.org/docs/latest/sql-ref-syntax-qry-select-window.html). We can also add additional options if necessary.

Reply:
Ok, I was looking at postgres for reference.  Since this would be an additive change (should be backwards compatible to add a new WINDOW_FRAME_...) I think we can leave this as it is.  We can add a new option later when we have a producer or consumer that supports it.